include/sdf/Filesystem.hh

```diff
@@ -128,6 +128,14 @@ namespace sdf
       IGN_UTILS_UNIQUE_IMPL_PTR(dataPtr)
     };
   }
+
+  /// \brief The source path replacement if it was parsed from a string,
```

Comment:
BTW Note that `std::string` has a non-trivial destructor (when SSO (short-string opt) ain't happening). Please note that this violates GSG:
https://ignitionrobotics.org/docs/all/contributing#style-guides
https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

See also: Drake's mitigation:
https://drake.mit.edu/code_style_guide.html
https://drake.mit.edu/styleguide/cppguide.html#Static_and_Global_Variables (`drake::never_destroyed<T>`)


This may be an existing defect, though.

Reply:
@azeey Can you advise whether this is important, or can be ignored for now?

Reply:
Oh thanks for pointing this out, I'll be mindful of it going forwards!